torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -222,6 +225,8 @@ void FunctionParameter::set_default_str(const std::string& str) {
     default_bool = (str == "True" || str == "true");
   } else if (type_ == ParameterType::DOUBLE) {
     default_double = atof(str.c_str());
+  } else if (type_ == ParameterType::COMPLEX) {
+    default_complex[0] = atof(str.c_str()); // TODO: parse "x + xj"?
```

Comment:
Don't you have to explicitly initialize `default_complex[1]` here too? Not that any of this is exercised at the moment.